packages/components/typography/src/utils/constants.js

```diff
@@ -4,7 +4,9 @@ export const colorTypes = {
 	DANGER: 'danger',
 	ERROR: 'error',
 	SUCCESS: 'success',
-	WARNING: 'warning'
+	WARNING: 'warning',
+	DISABLED: 'disabled',
+	DISABLED_ACTIVE: 'disabled-active'
```

Comment:
What is `disabled_active`? weird type

Reply:
For situations like menu, which can be active or checked and still disabled!